backend/models/discord_user.py

```diff
@@ -1,14 +1,16 @@
 import typing as t
 
-from pydantic import BaseModel
+from pydantic import BaseModel, validator
```

Comment:
Providing you agree with my comment on the validator, this import is not required.
```suggestion
from pydantic import BaseModel
```